examples/seismic/acoustic/acoustic_example.py

```diff
@@ -39,7 +39,7 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
     # Define receiver geometry (spread across x, just below surface)
     rec, u, summary = solver.forward(save=save, autotune=autotune)
 
-    if preset == 'constant':
+    if preset == 'constant-isotropic':
```

Comment:
whatta? it is true that we often allow hijacking small PRs to ship unrelated commits, but IMHO this PR is too critical for this kind of things

Reply:
sorry was debugging for oscar at same time forgot to revert